modules/data-store/src/data-store.js

```diff
@@ -57,7 +57,11 @@ class DataStore {
             id = entity.id; // eslint-disable-line prefer-destructuring
             realEntity = entity;
         }
-        const path = this[ENTITIES_BY_UUID][id].path; // eslint-disable-line prefer-destructuring
+        const localEntity = this[ENTITIES_BY_UUID][id];
+        if (!localEntity) {
+            return;
+        }
+        const path = localEntity.path; // eslint-disable-line prefer-destructuring
```

Comment:
Is there any reason to use `const path = localEntity.path;` instead of `const { path } = localEntity;` ?

Reply:
No particular reason, when this modules was developed the destructuring rule of eslint was ignored... Actually, if you check the file everytime eslint asks to use destructuring it is ignored (besides this PR).